File: php/features.php
<?php
declare(strict_types=1);

// CorporateBullshitGenerator SDK feature factory

require_once __DIR__ . '/feature/BaseFeature.php';
require_once __DIR__ . '/feature/TestFeature.php';


class CorporateBullshitGeneratorFeatures
{
    public static function make_feature(string $name)
    {
        switch ($name) {
            case "base":
                return new CorporateBullshitGeneratorBaseFeature();
            case "test":
                return new CorporateBullshitGeneratorTestFeature();
            default:
                return new CorporateBullshitGeneratorBaseFeature();
        }
    }
}

File: py/features.py
# CorporateBullshitGenerator SDK feature factory

from feature.base_feature import CorporateBullshitGeneratorBaseFeature
from feature.test_feature import CorporateBullshitGeneratorTestFeature


def _make_feature(name):
    features = {
        "base": lambda: CorporateBullshitGeneratorBaseFeature(),
        "test": lambda: CorporateBullshitGeneratorTestFeature(),
    }
    factory = features.get(name)
    if factory is not None:
        return factory()
    return features["base"]()
